src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -1589,6 +1592,52 @@ def use_radamsa_mutator_plugin(extra_env):
   return True
 
 
+def use_peach_mutator(extra_env, arguments):
+  """Decide whether or not to use peach mutator, and set up all of the
+  environment variables necessary to do so."""
+  # TODO(mpherman): Include architecture info in job definition and exclude
+  #  i386.
+  if environment.platform() != 'LINUX' or environment.get_value(
```

Comment:
nit: This line continuation looks a bit strange.

I'd do:

```python
if (environment.platform() != 'LINUX' or
    environment.get_value('MEMORY_TOOL') != 'ASAN'):
  ...

Reply:
Done

Reply:
why do we restrict peach mutator to this configuration only?

Reply:
Looks like you updated lines 1582-1584, but not here. We should move this into a shared function and call it from both places.

Reply:
Done